webrtc.html

```diff
@@ -10483,6 +10483,12 @@ <h2>Methods</h2>
               <p><code>Blob</code> object:</p>
               <p>Let <var>data</var> be the raw data represented by the
               <code>Blob</code> object.</p>
+              <div class="note" data-tests="RTCDataChannel-send-blob-order.html">
+              Although the actual retrieval of <var>data</var> from a <code>Blob</code> object
+              can happen asynchronously, the user agent will make sure to queue the data on
```

Comment:
I like this language, but I think we need to go further and use MUST here, since I don't see this enforced anywhere else.

If MUST in a note isn't ok then we should make it regular normative language (not a note). See my https://github.com/w3c/webrtc-pc/issues/2215#issuecomment-526667672.